### PR TITLE
Add an hostname_aliases parameter

### PR DESCRIPTION
**Pull request by npmeyer**
# Description

Adds an "aliases" parameter to the dokken driver that sets the same parameter in the Docker create request. This enables testing of cookbooks that depend on resolving multiple hostnames, such as cookbooks that configure multiple Apache virtual hosts.

## Issues Resolved

None.

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by npmeyer** [comment]
I'm not sure how to add a test for this. I'm not able to run the tests on the `master` branch locally (using `bundle exec kitchen test default-centos`); chef hangs in the test kitchen container while converging the execute[install gem bundle] resource.

**Comment by npmeyer** [comment]
Is there anything I can do to help get this PR reviewed and merged?

File: README.md [hunk 1]
```diff
@@ -558,6 +558,17 @@ driver:
...
 ```
 
+### Adding hostname aliases
+
+You can set the `aliases` parameter to create additional hostnames that will resolve to the container:
+
+```yaml
+driver:
+  name: dokken
+  aliases:
```
Comment (tas50) [inline README.md:568]: Can we rename this hostname_aliases so it's more clear what it achieves?

Comment (npmeyer) [reply README.md]: Changed!

**Comment by tas50** [comment]
Thanks. This will be very handy

**Pull request merged**